api/models/model.py

```diff
@@ -17,7 +19,7 @@
 import sqlalchemy as sa
 from flask import request
 from flask_login import UserMixin
-from sqlalchemy import Float, Index, PrimaryKeyConstraint, String, exists, func, select, text
+from sqlalchemy import Float, Index, PrimaryKeyConstraint, String, exists, func, text
```

Comment:
Lines 9 and 22 can be merged into a single line, just like the previous code. I'm also curious why this didn’t trigger autofix.